jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -1648,7 +1729,21 @@ def dot_product_attention(
     bias: Bias to be added to logits with a shape of BNTS.
     mask: Mask used to filter out logits with a shape of BNTS.
     q_seqlen: Non padded sequence length of Queries with a shape of B.
+      If q_offsets is set, q_seqlen should have shape [B,M] where M is the
+      maximum number of segments per batch. Fill the invalid entries with -1.
     kv_seqlen: Non padded sequence length of Keys and Values with a shape of B.
+      If kv_offsets is set, kv_seqlen should have shape [B,M] where M is the
+      maximum number of segments per batch. Fill the invalid entries with -1.
+    q_offsets: offset of each segment packed in Queries with a shape of [B,M+1]
```

Comment:
Should it be "query" instead of "Queries", since you are probably referring to the argument?